spec/support/betamocks.yml

```diff
@@ -37,6 +37,12 @@
     :cache_multiple_responses:
       :uid_location: body
       :uid_locator: 'root="2.16.840.1.113883.4.1" extension="(\d{9})"'
+  - :method: :post
+    :path: "/tithvitj"
+    :file_path: "multi/body/captures"
+    :cache_multiple_responses:
+      :uid_location: body
+      :uid_locator: '<v11:eMISservice(EpisodeRequest)>\n\s+<v12:edipiORicn>\n\s+<v13:edipiORicnValue>(\d+)'
```

Comment:
Definitely a gray area, but I see this more as two separate endpoints that deserve two separate configs.  For example, given an animal SOAP service:

```
GET /animals

# in request body
<AnimalType>Dog</AnimalType>

--- OR---

# in request body
<AnimalType>Cat</AnimalType>
```

it would have the following configurations:
```yaml
- :method: :get
  :path: /animals
  :file_path: "/animals/dogs"
  :cache_multiple_responses:
    :uid_location: body
    :uid_locator: '<AnimalType>Dog</AnimalType>'
- :method: :get
  :path: /animals
  :file_path: "/animals/cats"
  :cache_multiple_responses:
    :uid_location: body
    :uid_locator: '<AnimalType>Cat</AnimalType>'
```